torchvision/prototype/transforms/functional/_misc.py

```diff
@@ -8,7 +8,39 @@
 from torchvision.transforms import functional_tensor as _FT
 from torchvision.transforms.functional import pil_to_tensor, to_pil_image
 
-normalize_image_tensor = _FT.normalize
+
+def normalize_image_tensor(
+    image: torch.Tensor, mean: List[float], std: List[float], inplace: bool = False
+) -> torch.Tensor:
+    if not isinstance(image, torch.Tensor):
+        raise TypeError("Input img should be Tensor image")
+
+    if not image.is_floating_point():
+        raise TypeError(f"Input tensor should be a float tensor. Got {image.dtype}.")
+
+    if image.ndim < 3:
+        raise ValueError(
+            f"Expected tensor to be a tensor image of size (..., C, H, W). Got tensor.size() = {image.size()}"
+        )
+
+    if (isinstance(std, (tuple, list)) and not all(std)) or std == 0:
+        raise ValueError(f"std evaluated to zero after conversion to {image.dtype}, leading to division by zero.")
```

Comment:
The message needs at least to be adapted since there is no longer any conversion.